reactioncommerce/reaction
# style: update jsdoc to always use `returns` instead of `return`

Impact: **minor**  
Type: **style**

## Issue
We use both `return` and `returns` in our JSDoc comments. While both are valid, [`returns` is the official tag, and `return` is a `synonym` of it.](https://jsdoc.app/tags-returns.html)

## Solution
Update all tags to be `returns` to have consistent doc styling. While doing this, `eslint-changed-files` flagged some of the files to be fixed with other lint updates, so these were done as well. Also created https://github.com/reactioncommerce/reaction/issues/5449 to re-visit the `node/no-deprecated-api` rule.

## Breaking changes
None.

## Testing
1. Make sure the app starts, these are doc only changes so should have no affect on anything
1. See that 99% of the changes are simply adding an `s` to the end of returns, with the other 1% of changes small updates to satisfy the `eslint-changed-files` CI check
1. Make sure there are no eslint errors (CI check will confirm this)
1. See that there are ~85 `eslint` warnings remaining in the codebase

![style__update_jsdoc_to_always_use_`returns`_instead_of_`return`_by_kieckhafer_·_Pull_Request__5447_·_reactioncommerce_reaction](https://user-images.githubusercontent.com/4482263/62964091-2ccfee00-bdb7-11e9-80dd-db04d3ec68fe.png)
